### PR TITLE
runbooks, Add CnaoDown Alert 

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds the CnaoDown alert runbook

### CODE REVIEW

File: docs/runbooks/CnaoDown.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# CnaoDown
+
+## Meaning
+
+Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
```
[the remark this answers is not in the document]
Comment (RamLavi) [reply docs/runbooks/CnaoDown.md:5]: why not ? :) 
This runbook is meant for a sys-admin trying to figure out what's this pesky alert and why is it considered "warning" (and not "critical" for example).. I think a shot introduction is appropriate no? (but open for discussion of course)

[the remark this answers is not in the document]
Comment (sradco) [reply docs/runbooks/CnaoDown.md:5]: I think adding the explanation of what CNAO is , is important for users.

File: docs/runbooks/CnaoDown.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	kubectl -n $NAMESPACE get pods -l name=cluster-network-addons-operator
+	```
+ 
+## Mitigation
+
+There can be several reasons for CNAO's operator pod to be down, identify the root cause and fix it.
+
+- Check the CNAO's operator pod logs and status.
+	- `kubectl -n $NAMESPACE describe pods -l name=cluster-network-addons-operator`
+	- `kubectl -n $NAMESPACE logs -l name=cluster-network-addons-operator`
```
[the remark this answers is not in the document]
Comment (sradco) [reply docs/runbooks/CnaoDown.md:30]: I agree with @qinqon , please add to the Mitigation the steps to resolve the issue

[the remark this answers is not in the document]
Comment (RamLavi) [reply docs/runbooks/CnaoDown.md:30]: you are right.. then again real mitigation advice really depends on what broke there (did it go OOMKill? is there an internal bug?) 
However I think restarting pods is not going to work as well, as this alert is fired if the operator is down for more than 5 minutes.. this means that it's already stuck on crashLoopBackoff or something... 

Thinking more about this, maybe mitigation here means more like "sys-admin level" mitigation, so maybe the only thing is to gather logs and the like and call support.. @sradco what do you think?
looking at other runbooks ([VirtAPIDown](https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/VirtAPIDown.md) for example), do a same thing in mitigation..

[the remark this answers is not in the document]
Comment (qinqon) [reply docs/runbooks/CnaoDown.md:30]: What I mean is that gathering logs does not mitigate things, the state is going to be the same, not sure about the sepcific on what step with have to follow, but for sure mitigation steps has to end up fixing it or at least improving it.

[the remark this answers is not in the document]
Comment (RamLavi) [reply docs/runbooks/CnaoDown.md:30]: I agree with you both.. Now the only question remain is what _do_ I write in the mitigation section.. I will think about it some more..

[the remark this answers is not in the document]
Comment (RamLavi) [reply docs/runbooks/CnaoDown.md]: Done

**Review by qinqon — APPROVED** [review]

**Pull request merged**